### PR TITLE
Add new url for Q&A page, create new view, create new template with Q…

### PR DESCRIPTION
**Pull request by Katy600**
…&A content and add 2 new class to app.scss that remove spaces above and below title and paragraph.

### CODE REVIEW

File: mtp_bank_admin/apps/bank_admin/views.py [hunk 1]
```diff
@@ -63,3 +65,11 @@ def download_bank_statement(request, receipt_date):
     })
 
     return response
+
+class Q_And_A_View(TemplateView):
```
Comment (ibrechin) [inline mtp_bank_admin/apps/bank_admin/views.py]: Python code style is underscores pretty much everywhere except class names, which should be camel case i.e. QAndAView

File: mtp_bank_admin/apps/bank_admin/views.py [hunk 2]
```diff
@@ -63,3 +65,11 @@ def download_bank_statement(request, receipt_date):
...
+
+    template_name = "q_and_a_page/q_and_a_page.html"
+
+    
+
+
```
Comment (ibrechin) [inline mtp_bank_admin/apps/bank_admin/views.py]: Minor point, but you don't need this much space at the end of the file (or between the class name and the templae_name attribute) - one new line is enough.

File: mtp_bank_admin/apps/bank_admin/urls.py [hunk 3]
```diff
@@ -14,5 +16,6 @@
     url(r'^adi/download/$', views.download_adi_journal,
         name='download_adi_journal'),
     url(r'^bank_statement/download/$', views.download_bank_statement,
-        name='download_bank_statement')
+        name='download_bank_statement'),
+    url(r'^q_and_a_page$', Q_And_A_View.as_view(), name='q_and_a')
```
Comment (ibrechin) [inline mtp_bank_admin/apps/bank_admin/urls.py]: Probably just 'q_and_a' is enough as a path, the 'page' is a little redundant.

File: mtp_bank_admin/templates/q_and_a_page/q_and_a_page.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+	<p class="lede no-space-above" >When the prison business hub staff get your letter, they’ll send you a cheque.  </p><br>
+
+	<u><h1 class="heading-medium no-space-under-heading">If I send money using the service how will my data be used or stored?</h1></u>
+	<p class="lede no-space-under-heading">Go to: <a href="https://www.gov.uk/send-prisoner-money">www.gov.uk/send-prisoner-money</a></p>
+	<p class="lede no-space-above" >Click on ‘Terms and Conditions’ at the bottom of the page.</p><br>
+  </div>
+
+</div>
+{% endblock %}
+
```
Comment (ibrechin) [inline mtp_bank_admin/templates/q_and_a_page/q_and_a_page.html]: Generally you always end a file with a new line. The absence of one here is why github has put that little red sign there.

File: mtp_bank_admin/apps/bank_admin/urls.py (hunk 3 above)
Comment (ibrechin) [inline mtp_bank_admin/apps/bank_admin/urls.py]: Also the convention in Djnago is to always end paths with a forward slash

**Comment by ibrechin** [comment]
retest this please

**Comment by Katy600** [comment]
Thank you Ian. I am sorry I only just saw this! I have just made all those corrections.

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Pull request merged**